src/UpdatesLog.php

```diff
@@ -547,10 +558,6 @@ public function generateStatistics(
         $statistics['summary']['UNKNOWN'] += 1;
       }
 
-      if ($status === 'CURRENT') {
-        continue;
-      }
-
```

Comment:
This is THE change.
If you see other changes,
it means it is based on other non-merged branch.